docs/development/contributor_guidelines.rst

```diff
@@ -1,6 +1,12 @@
 Contributing Guidelines
 =======================
 
+Signing commits
+---------------
+
+Commits must be signed, as `explained in the GitHub documentation <https://help.github.com/articles/signing-commits-using-gpg/>`_.
+This helps verify commits proposed in a pull request are from a the expected author.
```

Comment:
My concern with this is that it further increases the technical bar for contributors and may deter contributions - how about we have this as a recommendation but not a requirement?

Reply:
Agreed on requirement—`s/must/should/` would suffice. Also, typo in "from a the": remove "a ".

Even if we enforce signing across the board for all commits, we're still losing full integrity by clicking the "merge" button on the GitHub interface, since the merge commit isn't signed. Still, for PR review, having commits marked as verified due to valid signatures is a Good Thing.™

Reply:
done